Projects/Foundations/Schemas/Docs-Topics/A/Alligator/Alligator-Data-Mine/alligator-data-mine-001-alligator-by-@blaaswe.json

```diff
@@ -7,34 +7,27 @@
         "updated": 1611661320849,
         "translations": [
             {
-                "language": "TR",
-                "text": "Özet: Bu veri madeni Williams Timsah (Alligator) göstergesinden oluşmaktadır",
-                "updated": 1644587038398
+                "language": "RU",
+                "text": "Этот источник данных состоит из индикатора Williams Alligator",
+                "updated": 1646896227169
```

Comment:
You are accidentally overwriting parts of the TR translation.

Reply:
I went through the files and I think I fixed everything.